packages/components/news/7196.breaking

```diff
@@ -0,0 +1 @@
+- Unify `Breadcrumbs` component implementations, remove custom BreadcrumbsPrimitive, since it's no longer needed. @sneridagh
```

Comment:
This is tricky grammar. I removed the last comma to associate the final clause with the preceding statement. Otherwise it implies the unification is no longer needed.

```suggestion
- Unify `Breadcrumbs` component implementations, while removing the custom `BreadcrumbsPrimitive` implementation since it's no longer needed. @sneridagh
```